.circleci/config.yml

```diff
@@ -1060,3 +1060,32 @@ workflows:
           name: integration_with_premium_latest
           requires:
             - build_premium
+      - acceptance_tests:
+          <<: *slack-fail-post-step
+          name: acceptance_with_premium_oldest
+          woo_core_version: 8.8.3
+          woo_subscriptions_version: 5.9.1
+          woo_memberships_version: 1.25.2
+          automate_woo_version: 5.8.5
+          codeception_image_version: 7.4-cli_20220605.0
+          wordpress_image_version: wp-6.4_php8.0_20231114.1
+          requires:
+            - build_premium
+      - unit_tests:
+          <<: *slack-fail-post-step
+          name: unit_with_premium_oldest
+          executor: wpcli_php_mysql_oldest
+          requires:
+            - build_premium
```

Comment:
Do we need this job? I mean, unit tests should test only one class or function with all other stuff mocked, so by this principle, they shouldn't be dependent on premium being active. I think that we don't even load WordPress for the unit tests.

Reply:
Good point. I will update it.